docs/modules/basic_speed_servo_info.rst

```diff
@@ -30,6 +30,7 @@ Supported IQUART Clients
         * :ref:`arming_handler`
         * :ref:`stopping_handler`
         * :ref:`iquart_flight_controller_interface`
+        * :ref:`throttle_source_manager`
```

Comment:
We don't actually have a basic speed release out yet that supports this, so maybe not yet the time to add it here